phpstan.neon

```diff
@@ -478,3 +478,9 @@ parameters:
                 - packages/git-wrapper/src
                 - packages/set-config-resolver/src/ConfigResolver.php
                 - packages/set-config-resolver/src/SetAwareConfigResolver.php
+
+        # needed as `humbug/php-scoper` provides `Isolated\Symfony\Component\Finder\Finder`
```

Comment:
I realize I took a liberty here, but I was unable to find a way to address `humbug/php-scoper`'s [expected `Isolated\Symfony\Component\Finder\Finder` usage](https://github.com/humbug/php-scoper#configuration).